docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,365 @@
+# Autofill
+
+Autofill is a feature that allows you to fill out commonly-entered information such as email, account and address in a text input field. email, account and address in a text input field.
+Using autofill functions, you can get the retrieve the data previously entered by the user and stored or request to store data in an autofill service.
+
+The main features of the Autofill API include:
+
+- [Requesting to get the presence of autofill data and autofill authentication information](#requesting-to-get-the-presence-of-autofill-data-and-autofill-authentication-information)
+
+  You can get the autofill service information such as autofill service name, service message and service logo image to use `autofill_auth_info_request()`.
+  You can get the presence of autofill data for each view in autofill service information to be received.
+
+- [Retrieving the data previously entered by the user and stored in an autofill service](#retrieving-the-data-previously-entered-by-the-user-and-stored-in-an-autofill-service)
+
+  Use the `autofill_fill_request()` function to retrieve the data previously entered by the user and stored in an autofill service
+
+- [Requesting to store the data entered by the user in autofill service](#requesting-to-store-the-data-entered-by-the-user-in-autofill-service)
+
+  Use the `autofill_commit()` function to store the data entered by the user in autofill service
+
+## Prerequisites
+
+1. Configure autofill service:
+
+   Set autofill service in Settings > Language and input > Input Assistance > Autofill setting.
+   In some environment, the autofill service can be designated according to the system configuration.
+
+2. To use the autofill API (in [mobile](../../api/mobile/latest/group__CAPI__UIX__AUTOFILL__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__UIX__AUTOFILL__MODULE.html) applications), include the `<autofill.h>` header file in your application:
+
+   ```c
+   #include <autofill.h>
+   ```
+
+3. To use the autofill library, create an autofill handle:
+
+   The autofill handle is used in other autofill functions as a parameter.
+
+   ```c
+   void
+   create_autofill_handle()
+   {
+       autofill_h ah;
+       int ret;
+       ret = autofill_create(&ah);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+4. When you no longer need the autofill library, destroy the autofill handle using the `autofill_destroy()` function:
+
+   ```c
+   void
+   destroy_autofill_handle(autofill_h ah)
+   {
+       int ret;
+
+       ret = autofill_destroy(ah); /* ah is the autofill handle */
+       if (ret != AUTOFILL_ERROR_NONEt)
+           /* Error handling */
+   }
+   ```
+
+   > **Note**
+   >
+   > Do not use the `autofill_destroy()` function in a callback function.
```

Comment:
It is not recommended to use the `autofill_destroy()` function in a callback function.